packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -199,6 +205,43 @@ enum ListTileControlAffinity {
 /// ```
 /// {@end-tool}
 ///
+/// For accessibility purposes, tappable leading and trailing widgets have to
+/// be at least 48x48px in size. However, to adhere with Material spec,
```

Comment:
I think it would actually be clearer to leave out the "px" unit here and below. All of our dimensions are in logical pixel units and we generally don't include "px".